fastlane/spec/actions_specs/import_certificate_spec.rb

```diff
@@ -11,7 +11,7 @@
         password = 'testpassword'
 
         keychain_path = File.expand_path(File.join('~', 'Library', 'Keychains', keychain))
-        expected_command = "security import #{cert_name} -k '#{keychain_path}' -P #{password} -T /usr/bin/codesign -T /usr/bin/security 1> /dev/null"
+        expected_command = "security import #{cert_name} -k #{keychain_path} -P #{password} -T /usr/bin/codesign -T /usr/bin/security 1> /dev/null"
```

Comment:
isn't this missing `.shellescape` then?

Reply:
Escaping is a no-op here because there are no special characters in the test parameters. 
Good catch and the other parameters should be escaped too.